doc/changes/changes_5.0.0.md

```diff
@@ -0,0 +1,33 @@
+# Exasol Virtual Schema Common 5.0.0, released 2020-??-??
```

Comment:
```suggestion
# Exasol Virtual Schema Common 5.0.0, released 2021-??-??
```

Reply:
It will be updated automatically if I release